src/qml/AboutPage.qml

```diff
@@ -25,18 +25,18 @@ import org.thetailcompany.digitail 1.0
 Kirigami.AboutPage {
     objectName: "aboutPage";
     aboutData: {
-        "displayName" : "DIGITAiL Controller",
+        "displayName" : "DIGITAiL App",
         "productName" : "digitail/controller",
         "programLogo" : ":/icon-round.png",
         "componentName" : "digitail",
         "shortDescription" : "DIGITAiL designed and created by The Tail Company",
         "homepage" : "https://thetailcompany.com/",
         "bugAddress" : "info@thetailcompany.com",
-        "version" : "1.0",
+        "version" : "v1.0",
         "otherText" : BTConnectionManager.isConnected ? qsTr("Connected tail is version %1").arg(BTConnectionManager.tailVersion) : "",
         "authors" : [
             {
-                "name" : "Dan Leinir Turthra Jensen\n",
```

Comment:
The reason for doing this is to avoid overflow caused by a bug in the Kirigami AboutPage implementation. Without the newline (which absolutely is a workaround hack), it looks as seen in the attached screenshot.
![image](https://user-images.githubusercontent.com/5261787/53344979-d4da2380-390b-11e9-92e8-b974622a55d5.png)

Reply:
I see that, so, another way is to have an extra space at the left side before the email, because Kirigami hardcoded the space:

```
text: qsTr("%1 <%2>").arg(modelData.name).arg(modelData.emailAddress);
```

In this case I think the extra space before the email is a less evil ;) or we can patch the Kirigami

Reply:
Also I have tried to add `\b` (backspace) to email string in hope it removes the space, but QML just shows a symbol for `backspace`

Reply:
Rather than outright patching (i really, really don't want us to be maintaining a patch set for an open source component of an open source app), i would very much prefer the Kirigami issue was fixed. The issue is, quite annoyingly, deep inside the Card component (the About page simply shows it happening). I did try a quick fix on it, but Marco wasn't too keen on that: https://phabricator.kde.org/D18180